### PR TITLE
Change user password

### PR DESCRIPTION
**Pull request by Las6103**
- Related Issue (include '#'): #425 

- Description of changes made:
This branch contains Chris's social media adjustments as well as the password change feature.
- Is the feature complete/bug resolved/etc..:
The password feature is complete and fully functional.
- Any known bugs/strange behavior:
NA
- Is there specific feedback you would like on these changes:
- Styling could be adjusted if necessary
- Screenshot(s):

### CODE REVIEW

File: client/src/components/Bio/Bio.css [hunk 1]
```diff
@@ -44,3 +52,7 @@
   max-width: 100%;
   flex-wrap: wrap;
 }
+
+.password_header {
+  color: #6a4c93;
+}
```
Comment (0xZakk) [inline client/src/components/Bio/Bio.css]: Can you use the CSS naming convention?

**Review by 0xZakk — CHANGES_REQUESTED** [review]
Rather than have an alert pop down, can you display a message in UI? If the password is successfully changed, we should replace the form with a success message. If there's an error, then display that error with the form.

Also, can we do it without the email field? You can pull the email from the current user who is signed in. You want to check the JWT to verify auth anyway, rather than effectively have them re-sign in.

**Review by 0xZakk — CHANGES_REQUESTED** [review]
Add labels to inputs and then merge it

**Comment by Las6103** [comment]
Merging the request. No conflicts.

**Pull request merged**